### PR TITLE
Feature 16 survey waste material

### PR DESCRIPTION
**Pull request by jarinykanen**
Resolves: #16

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -2099,5 +2580,53 @@ components:
...
+        metadata:
+          $ref: "#/components/schemas/Metadata"
+
+    Usage:
+      type: object
+      description: Waste usage object
+      required:
+        - componentName
+        - reusableMaterialId
+        - usability
```
Comment (anttileppa) [inline swagger.yaml:2592]: Required fields do not match actual fields

File: swagger.yaml [hunk 2]
```diff
@@ -2099,5 +2580,53 @@ components:
...
+        name:
+          type: string
+          description: Usage name
+        metadata:
+          $ref: "#/components/schemas/Metadata"
+
+    Waste:
+      type: object
+      description: Waste construction material object
+      required:
```
Comment (anttileppa) [inline swagger.yaml:2607]: Required fields do not match actual fields

**Review by anttileppa — CHANGES_REQUESTED** [review]

File: swagger.yaml [hunk 3]
```diff
@@ -2099,5 +2581,53 @@ components:
...
+          format: uuid
+          description: Waste material ID
+        usageId:
+          type: string
+          format: uuid
+          description: Usage ID
+        amount:
+          type: number
+          format: double
+          description: Amount of reusable material (tons)
```
Comment (jarinykanen) [inline swagger.yaml:2628]:
```suggestion
          description: Amount of waste material (tons)
```

File: swagger.yaml [hunk 4]
```diff
@@ -2099,5 +2581,53 @@ components:
...
+        description:
+          type: string
+          description: More detailed description of the reusable material
```
Comment (jarinykanen) [inline swagger.yaml:2631]:
```suggestion
          description: More detailed description of the waste material
```

**Pull request merged**